examples/single_zone_cluster/main.tf

```diff
@@ -35,25 +35,30 @@ locals {
   }
   sz_pool = [
     {
-      subnet_prefix    = "default"
-      pool_name        = "default" # ibm_container_vpc_cluster automatically names default pool "default" (See https://github.com/IBM-Cloud/terraform-provider-ibm/issues/2849)
-      machine_type     = "bx2.4x16"
-      workers_per_zone = 2
+      subnet_prefix      = "default"
+      pool_name          = "default" # ibm_container_vpc_cluster automatically names default pool "default" (See https://github.com/IBM-Cloud/terraform-provider-ibm/issues/2849)
+      machine_type       = "bx2.4x16"
+      workers_per_zone   = 2
+      minSize            = 1
+      maxSize            = 5
+      enable_autoscaling = true
```

Comment:
This looks like the single zone example is enabling auto scaling on the default worker pool. With the ocp_base module setting both  `enable_autoscaling = true` and `ignore_worker_pool_size_changes = true`.

The Kubernetes or time provider provider are not included. These would definitely be required since the plan is to use the new resources.